src/ol/renderer/canvas/vectortilelayer.js

```diff
@@ -413,12 +412,12 @@ ol.renderer.canvas.VectorTileLayer.prototype.renderFeature = function(feature, s
     for (var i = 0, ii = styles.length; i < ii; ++i) {
       loading = ol.renderer.vector.renderFeature(
           replayGroup, feature, styles[i], squaredTolerance,
-          this.handleStyleImageChange_, this) || loading;
+          this.handleStyleImageChange_.bind(this)) || loading;
     }
   } else {
     loading = ol.renderer.vector.renderFeature(
         replayGroup, feature, styles, squaredTolerance,
-        this.handleStyleImageChange_, this) || loading;
+        this.handleStyleImageChange_.bind(this)) || loading;
```

Comment:
This has the problem that a closure is created every time it is called. Alternatively it is possible to store a bound version of the method and use that one.